verilog/analysis/verilog_linter.cc

```diff
@@ -262,7 +266,7 @@ LinterConfiguration LinterConfigurationFromFlags() {
   config.UseRuleBundle(rules);
 
   // Apply external waivers
-  config.external_waivers = absl::GetFlag(FLAGS_waivers_file);
+  config.external_waivers = absl::GetFlag(FLAGS_waiver_files);
```

Comment:
This will move with your other PR, right?

Reply:
Yes, either this or the other PR will have to be rebased after one of them is merged.